package.json

```diff
@@ -18,46 +18,45 @@
   "engines": {
     "node": ">=12.18.0"
   },
-  "dependencies": {
-    "date-fns": "^2.27.0"
-  },
+  "dependencies": {},
```

Comment:
This change is a breaking change because the user must install `date-fns` manually. 

I'm ok to do major, it's not a big deal. I would also increase minimum NodeJS version to 14.18 because we will soon upgrade app * vis

Reply:
@moroine Okay, I can rename the commit and force push. 👍🏻